### PR TITLE
NO-ISSUE: add developer review dashboard tools

### PR DESCRIPTION
**Pull request by dhellmann**
Add script and docs with demo installation of dinghy for building a
review dashboard based on PRs related to MicroShift across different
repositories.

### CODE REVIEW

File: scripts/review-dashboard/dinghy.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - title: Code Changes
+        url: https://github.com/openshift/microshift
+
+      # Include changes to MicroShift CI
+      - title: CI Changes
+        search: ( repo:openshift/origin OR repo:openshift/release ) is:pr is:open microshift
+
+      # Look for documentation changes for MicroShift
+      - title: Documentation
+        search: repo:openshift/openshift-docs is:pr is:open microshift
```
Comment (pacevedom) [inline scripts/review-dashboard/dinghy.yaml:26]: I would include `microshift OR ushift` in the search query, just in case.

**Review by pacevedom — COMMENTED** [review]
Nice!

File: scripts/review-dashboard/dinghy.yaml (hunk 1 above)
Comment (dhellmann) [reply scripts/review-dashboard/dinghy.yaml]: Good point, done.

**Comment by ggiguash** [comment]
This is great! Thank you @dhellmann

**Pull request merged**